src/main/java/io/jenkins/plugins/ml/IPythonKernelInterpreter.java

```diff
@@ -0,0 +1,21 @@
+package io.jenkins.plugins.ml;
+
+public class IPythonKernelInterpreter implements KernelInterpreter {
+
+    /**
+     * IPython will be connected and interpreted through the object of this class.
+     * @param code - python code to be executed
+     * @return null
```

Comment:
Instead of `@return null`, I think it would be better to define what is supposed to be returned. The type is a String, so I think that would be something like `@return the result of the interpreted code` or something similar?

Reply:
yh that would be nice.